pkg/controller/upgradeconfig/upgradeconfig_controller.go

```diff
@@ -117,19 +122,56 @@ func (r *ReconcileUpgradeConfig) Reconcile(request reconcile.Request) (reconcile
 	}
 
 	status := history.Phase
-	reqLogger.Info("current cluster status", "status", status)
+
+	// status should never be an empty string at this stage. Explicity test and return error if it is.
+	if status == "" {
```

Comment:
Is this necessary? Cant see how it will be empty

Reply:
https://github.com/openshift/managed-upgrade-operator/pull/71/files/ad2e4705621ded503b2b5a67cd942055fff39b2c#diff-416322dadc9a204248e5e54f38904009L123

We currently accept it as valid. Id prefer to handle this explicitly.

Reply:
I'd remove that too :joy: